bedrock/privacy/templates/privacy/base-quantum.html

```diff
@@ -2,6 +2,7 @@
  # License, v. 2.0. If a copy of the MPL was not distributed with this
  # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
 
+{% add_lang_files "privacy/index" %}
```

Comment:
This seems to work ok, but looking in this template there are other strings here already e.g. `"Choose how you want to share this data in Firefox."`, which for some reason live in `.main.lang`. Just want to make sure we're still ok using this older lang file for these newer templates. Thoughts @flodolo ?

Reply:
Redirecting to @peiying2

Reply:
@peiying2 I'm not quite clear if this means we are ok to use the older `privacy/index.lang` file, or not?

Reply:
@alexgibson yes, let's use the older template privacy/index.lang to keep it simple. Thx.

Reply:
Thanks!